packages/doc/out/index.html

```diff
@@ -76,6 +76,6 @@ <h2>Get started</h2>
         </div>
       </footer>
     </div>
-    <script src="./index.5edce124c07655354b00.js"></script>
```

Comment:
This surprises me that changing `page.js` would change `contenthash` of client side JavaScript.